data/core/macros/special-notes.cfg

```diff
@@ -126,171 +132,115 @@ _"This unit has a defense cap on certain terrain types — it cannot achieve a h
-        note={INTERNAL:SPECIAL_NOTES_AMBUSH}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_NIGHTSTALK
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_NIGHTSTALK}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_CONCEALMENT
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_CONCEALMENT}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_SUBMERGE
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_SUBMERGE}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_FEEDING
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_FEEDING}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_BERSERK
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_BERSERK}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_BACKSTAB
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_BACKSTAB}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_PLAGUE
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_PLAGUE}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_SLOW
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_SLOW}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_PETRIFY
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_PETRIFY}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_MARKSMAN
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_MARKSMAN}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_MAGICAL
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_MAGICAL}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_SWARM
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_SWARM}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_CHARGE
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_CHARGE}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_DRAIN
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_DRAIN}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_FIRSTSTRIKE
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_FIRSTSTRIKE}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
 #enddef
 
 #define NOTE_POISON
-    [special_note]
-        note={INTERNAL:SPECIAL_NOTES_POISON}
-    [/special_note]
+#deprecated 3 1.17 This note is now added automatically.
```

Comment:
I think this is actually `DEP_LEVEL::REMOVED` with a post-removal stub instead of `DEP_LEVEL::FOR_REMOVAL`.

Reply:
So you're saying I should change it from 3 to 4?

Reply:
I think so.